safe_transaction_service/tokens/services/price_service.py

```diff
@@ -46,6 +47,14 @@
 )
 from ..tasks import EthValueWithTimestamp, calculate_token_eth_price_task
 
+
+class MistswapOracle(UniswapV2Oracle):
```

Comment:
Oracles should go in https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/oracles/oracles.py